### ISSUE TITLE
Suggestion: Balancing of Leather Working and Farming against other gathering "jobs"

### ISSUE THREAD

**Issue by Angelnoir** [anegostudios/VintageStory-Issues#378]
It feels hugely unbalanced how fast you are able to progress in certain areas of crafting compared to others.
In Mining you can advance pretty fast. You can find any of the numerous surface copper ores, or could get enough for a pickaxe from panning in feasible time. With clay in reach you are in copper age in about an ingame week. With prospecting and a bit of luck you advance to "bronze age" in about another week and to iron age in an additional week. Its quite linear. 

If you start Farming early on, it takes a really long time to gather enough Seeds and to learn how to properly protecting your fields against hares. Not to mention that tera preta is quite hard to find too. Then you have to wait, crop rotate accordingly, hope for some extra seeds (chance is too low imho) while harvesting and start over. It takes literally ages to get enough flax to craft things like armor, wind mill sails or even smaller items that require linen.

In Leather Working it also take ages to produce some leather, because you cannot start before reaching copper age and you have to wait an ingame week to get your first batch. (If you're lucky to find lime and oak somewhere close).
In my current game we (two players) focused on everything else than mining and already progressed to iron age while only produced 1 leather (first trial batch) and 4 linen.

While every game and seed is different in terms of finding the right ores for progressing, the time you need to progress in leather working and farming is mostly fixed. 

tl:dr 
```
Finally here are my suggestions for balancing:
- increase chance of additional seeds while harvesting
- slightly decrease chance of gaining copper through panning
- decrease time needed for tanning
- switch oak logs for tanning with planks and reduce effective amount of logs needed per hide
- make tera preta more visible (maybe colored on map / minimap just like the different stone types)
- Adding additional fertilizers (maybe crafting all-round fertilizer (15% from each) from rotted food in addition to create compost blocks of it) 
- slightly increase number of wild farming plants 

```

**Comment by Craluminum2413** [comment]
I think it would be nice to make a nitrogen rich fertilizer from feathers

**Comment by sirStroam** [comment]
No longer supporting suggestions/enhancements here. Please move to forum suggestions.
https://www.vintagestory.at/forums/forum/5-suggestions/
